src/components/error-page/error-page-slide.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { CopyToClipboard } from "react-copy-to-clipboard";
```

Comment:
Is copy-to-clipboard available in preact?  

No need to worry about this now but we are hoping to switch to preact instead of react.